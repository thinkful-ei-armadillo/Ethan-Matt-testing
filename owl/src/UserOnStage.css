.userOnStage {
  background-color: #424854;
}
