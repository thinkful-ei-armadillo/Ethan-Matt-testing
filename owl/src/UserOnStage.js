import React from 'react';

import './UserOnStage.css';

function UserOnStage(props){

  return(
    <div className="userOnStage">
    <div className='userName'>
        <h6 className="userOnStage-name" >{props.name}</h6>
      </div>
      <img className="userOnStage-avatar" src={props.avatar} alt={props.name} />
      
    </div>
  );
}

export default UserOnStage;
