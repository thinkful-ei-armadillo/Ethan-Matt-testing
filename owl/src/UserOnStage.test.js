import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import UserOnStage from './UserOnStage';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<UserOnStage
    key={42}
    name={'alice'}
    avatar={'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1'}
  />, div);
  ReactDOM.unmountComponentAtNode(div);
});


describe('Snapshot tests', () => {

  it('renders the UI as expected', () => {
    const tree = renderer
    .create(<UserOnStage
      key={42}
      name={'alice'}
      avatar={'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1'}
    />)
    .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders the UI as expected', () => {
    const tree = renderer
    .create(<UserOnStage
      key={42}
      name={'alice'}
      avatar={'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1'}
    />)
    .toJSON();
    expect(tree).toMatchSnapshot();
  });

});
