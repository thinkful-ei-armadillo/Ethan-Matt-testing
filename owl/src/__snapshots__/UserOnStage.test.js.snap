// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot tests renders the UI as expected 1`] = `
<div
  className="userOnStage"
>
  <img
    alt="alice"
    className="userOnStage-avatar"
    src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
  />
  <div>
    <h6
      className="userOnStage-name"
    >
      alice
    </h6>
  </div>
</div>
`;

exports[`Snapshot tests renders the UI as expected 2`] = `
<div
  className="userOnStage"
>
  <img
    alt="alice"
    className="userOnStage-avatar"
    src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
  />
  <div>
    <h6
      className="userOnStage-name"
    >
      alice
    </h6>
  </div>
</div>
`;
